app/service/plugin/PluginMetadata.php

```diff
@@ -51,13 +49,14 @@ public static function parseFromFile(string $file): ?self
         }
         $head = @fread($fp, 8192);
         @fclose($fp);
-        if (!is_string($head) || $head === '') {
+        if (!is_string($head)) {
             return null;
         }
 
         // 仅解析文件顶部第一个块注释
         if (!preg_match('#/\*\*(.*?)\*/#s', $head, $m)) {
-            return null;
+            // 兼容：若主文件没有头注释，依然尝试读取 plugin.json
```

Comment:
增强了插件元数据解析逻辑，现在即使主文件没有头部注释，也会尝试读取plugin.json文件。这提高了插件系统的兼容性和灵活性。

```suggestion
// 仅解析文件顶部第一个块注释
if (!preg_match('#/\*(.*?)\*/#s', $head, $m)) {
    // 兼容：若主文件没有头注释，依然尝试读取 plugin.json
    $m = [1 => ''];
}